README.md

```diff
@@ -17,10 +17,7 @@ Package versions produced by official builds for this repo will automatically co
 
 ## How to service a library
 
-The default build will automatically build all of the libraries that have been added to the repo. That said, given this repository is used to service packages, and you won't always want to service all of the packages at the same time, packages for libraries are not built by default. Whenever a package needs to be serviced, the following steps will need to be follwed:
-
-1. Determine the ServiceVersion to be used. When servicing a fix on a library package, the ServiceVersion (third part of the NuGet package version) will need to be bumped. This property is found in the library's src project. It's also possible that the property is not in that file, in which case you'll need to add it to the project and set it to 1. If the property is already in the source project, then just increment the servicing version by 1.
-2. Ensure that the source project doesn't have the property `<IsPackable>false</IsPackable>` set. This will ensure that the package for this library will be produced as part of the build.
+The default build will automatically build all of the libraries that have been added to the repo. That said, given this repository is used to service packages, and you won't always want to service all of the packages at the same time, packages for libraries are not generated by default. To enable a library to produce a package and be serviced, set the `<IsPackable>` property to `true` in the library's project file.
 
```

Comment:
You need to talk about versioning here.  What do libraries do each time they ship?

Reply:
In the event of a breaking change (= dropping TFMs), the package version's minor digit should be incremented. For all other cases a developer shouldn't need to change any of the versions, as the bot will take care of that.

Reply:
I see - you'd like to automate incrementing.  That's a noble goal - I didn't realize that was something we cared to do for this repo where we don't expect much churn.